README.md

```diff
@@ -72,13 +72,14 @@ The following commands are supported:
 
   Begin a build.
 
-* `acbuild end ACI_PATH`
+* `acbuild end`
 
-  End a build, writing the resulting ACI to `ACI_PATH`.
+  End a build, deleting the current build context.
 
-* `acbuild abort`
+* `acbuild write ACI_PATH`
```

Comment:
Nit, I suggest listing `write` before `end` so it's more obvious that the normal workflow involves both.